### PR TITLE
LibJS: The Math.ceil() of a number between -1 and 0 should be -0, according to the spec.

### PR DESCRIPTION
**Pull request by qeeg**

### CODE REVIEW

File: Libraries/LibJS/Runtime/MathObject.cpp [hunk 1]
```diff
@@ -125,10 +125,13 @@ JS_DEFINE_NATIVE_FUNCTION(MathObject::floor)
 JS_DEFINE_NATIVE_FUNCTION(MathObject::ceil)
 {
     auto number = interpreter.argument(0).to_number(interpreter);
+    auto number_double = number.as_double();
     if (interpreter.exception())
         return {};
     if (number.is_nan())
         return js_nan();
+    if (number_double < 0 && number_double > -1)
+        return Value(-0.f);
```
Comment (linusg) [inline Libraries/LibJS/Runtime/MathObject.cpp:134]:
```suggestion
        return Value(-0);
```

Comment (qeeg) [reply Libraries/LibJS/Runtime/MathObject.cpp:134]: The test doesn't work if I change it to that.

Comment (linusg) [reply Libraries/LibJS/Runtime/MathObject.cpp:134]: huh - `-0.0` then? Either way, there's no float overload of `JS::Value`.

File: Libraries/LibJS/Runtime/MathObject.cpp [hunk 2]
```diff
@@ -125,10 +125,13 @@ JS_DEFINE_NATIVE_FUNCTION(MathObject::floor)
 JS_DEFINE_NATIVE_FUNCTION(MathObject::ceil)
 {
     auto number = interpreter.argument(0).to_number(interpreter);
+    auto number_double = number.as_double();
```
Comment (linusg) [inline Libraries/LibJS/Runtime/MathObject.cpp:128]: This needs to be after the exception check below as `to_number()` can return an empty value (if it throws), and `as_double()` would assert in that case.

**Pull request merged**